scripts/bundleSize.sh

```diff
@@ -3,7 +3,7 @@
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
 min=498
-max=517
+max=519
```

Comment:
When we change these, both should change. Can you make sure they are 5K +/- the smallest & biggest services please

Reply:
Having chatted to @dr3 @rhenshaw56 please can you update the comment to clarify the correct process on this PR, something like:

```
# Size limit for all bundles used by each service (K) - update frequently!
#
# If updating:
# - The min value should be 5K less than the smallest service bundle and max should be 5K greater than the largest bundle
# - Bundle sizes can be viewed by running a production build
```